content/en/blog/_posts/2024-02-21-prevent-unauthorized-volume-mode-conversion-ga.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: blog
+title: "Kubernetes 1.30: Preventing unauthorised volume mode conversion moves to GA"
+date: 2024-03-22
```

Comment:
The planned publication date for this blog is `2024-04-30`

Reply:
Fixed